src/Security/OpenIdConfigurationProvider.php

```diff
@@ -192,10 +197,32 @@ public function getAuthorizationUrl(array $options = []): string
      *
      * @throws ItkOpenIdConnectException
      */
-    public function validateIdToken(string $idToken, string $nonce): object
+
+    /**
+     * Do any required verification of the id token and return an array of decoded claims
+     *
+     * @param string $idToken
+     *   Raw id token as string
+     *
+     * @param string $nonce
+     *   Nonce as string
+     *
+     * @param int $leeway
+     *   Leeway set in seconds. Defaults to 0 and must be positive
+     *
+     * @return object
```

Comment:
Why remove the description: "The JWT's payload as a PHP object"

Reply:
Fixed in 860c85a